README.md

```diff
@@ -112,6 +112,7 @@ python manage.py resources_import --all kirjastot
 Settings are done either by setting environment variables named after the setting or adding them to a `.env` file in the project root. The .env file syntax is similar to TOML files (INI files), ie. key-value pairs. The project root is the directory where this README is found. You can also set settings in a local_settings.py, which allows you to set any variables whatsoever. However, some of the settings documented here are named differently in settings.py, especially authentication variables.
 
 - `DEBUG`: Whether to run Django in debug mode. [Django setting](https://docs.djangoproject.com/en/2.2/ref/settings/#debug).
+- `DEBUG_TOOLBAR`: Whether to enable Django Debug Toolbar. [Django Debug Toolbar](https://django-debug-toolbar.readthedocs.io/en/latest/index.html).
```

Comment:
I think you also need to set INTERNAL_IPS correctly (i.e. to 127.0.0.1) when debug toolbar is enabled - INTERNAL_IPS is mentioned in the README but might not be clear that it needs to be set. It might be worth mentioning that point here, it's a common stumbling block when setting up debug toolbar.

Reply:
It might also be good to mention that `DEBUG` has to be `True` as well.

Reply:
`INTERNAL_IPS` is actually set to ['127.0.0.1'] by default in the settings when `DEBUG` is enabled. I'll add a mention about `DEBUG`

Reply:
Fixed